src/game/default/g_ballistics.c

```diff
@@ -315,6 +315,69 @@ static void G_GrenadeProjectile_Explode(g_entity_t *self) {
 			G_BurnMark(self->s.origin, plane, surf, 20);
 		}
 	}
+	
+	// attempt to stop the sound playing...doesnt work
+	self->s.sound = 0;
```

Comment:
Maybe, I shortened the click sound to a single click rather than 3 seconds worth. Seems to work.